src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -639,6 +628,166 @@ save_availableCaptureId > (testExpressionCaptureId = ((IFlowCaptureReferenceOper
             return new FlowCaptureReference(resultCaptureId, operation.Syntax, operation.Type, operation.ConstantValue);
         }
 
+        private static bool IsNullableType(ITypeSymbol typeOpt)
```

Comment:
There's also such a method in `SymbolDisplayVisitor`, a very similar one (without nullable handling) in `TypeSymbolExtensions` and maybe some others too.
Consider moving this to a common implementation.

```c#
        private static bool IsNullableType(INamedTypeSymbol type)
        {
            var original = type.OriginalDefinition;
            return original != null && original.SpecialType == SpecialType.System_Nullable_T;
        }
```

Reply:
> Consider moving this to a common implementation

I'll do this in a separate PR, if that is OK